identity-composer-component.html

```diff
@@ -1,31 +1,137 @@
+    position: absolute;
+    margin-left: 50%;
+    margin-top: 1%;
+  }
+}
+
+.card-wrapper {
+  position: relative;
+
+}
+
+.info-icon-element {
+
+  position: relative;
+  color: rgba(0, 0, 0, 0.85);
+  right: 1.5%;
+}
+
+.icon-wrapper {
+  right: 0%;
+  top: 2.94%;
+  z-index: 2;
+  background-color: rgba(255, 255, 255, 0.6);
+  position: absolute;
+  width: 20px;
+  height: 20px;
+  border-radius: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.br-identity-profile-container {
+  padding-top: 3.5%;
+}
+
+@media (max-width: 767px) {
+  .br-identity-profile-container {
+    padding-top: 6.5%;
+  }
+
+
+}
+.modal-title {
+  font-size: 26px;
+}
+
+</style>
+
+
 <div ng-if="$ctrl.loading" style="width: 100vw; font-size: x-small;">
   <i class="fa fa-refresh fa-spin text-muted"></i> Building profiles...
 </div>
 <div ng-if="!$ctrl.loading" ng-style="$ctrl.profileContainerStyle">
   <div ng-if="!!$ctrl.profiles.length" class="br-identity-profile-list"
     ng-repeat="profile in $ctrl.profiles">
-    <h4 ng-mouseenter="$hover=true" ng-mouseleave="$hover=false"
-      ng-click="$ctrl.select(profile, $index)">
+<!--     <h4 ng-mouseenter="$hover=true" ng-mouseleave="$hover=false">
       <span class="label label-primary" style="cursor: pointer;">
+        
+      </span>
+    </h4> -->
+ 
+    <div class="text-container">
+      <div class="text-element"
+        style="font-size: calc({{$ctrl.credentialWidth.select}} * 0.07)">
         Select Profile {{$index + 1}}
         <i ng-if="$ctrl.sending && $ctrl.selectedIndex === $index"
-          class="fa fa-refresh fa-spin"></i>
-      </span>
-    </h4>
-    <div class="br-identity-profile-container"
+            class="fa fa-refresh fa-spin"></i>        
+      </div>        
+    </div>
+    <div class="br-identity-profile-container" 
       ng-mouseenter="$hover=true" ng-mouseleave="$hover=false"
-      ng-class="{'br-identity-profile-container-hover': $hover}">
+      ng-class="{'br-identity-profile-container-hover': $hover}"
+      ng-click="$ctrl.select(profile, $index)">
+
       <div class="br-identity-profile-item"
-        ng-repeat="credential in profile.credentials"
-        ng-click="$ctrl.credential=credential; $ctrl.showCredentialDetails=true">
+        ng-repeat="credential in profile.credentials">
         <!-- TODO: ng-class="alert alert-warning"/"alert-danger for superfluous
           and sensitive properties -->
         <!-- TODO: change simple to complex displayer -->
-        <br-credential br-credential="credential" br-show-actions="false"
-          br-options="{displayer: {style: {width: $ctrl.credentialWidth.select}}}">
-        </br-credential>
+        <div class="card-wrapper" style="width: {{$ctrl.credentialWidth.select}}">
+          <div class="icon-wrapper"
+              style="width: calc({{$ctrl.credentialWidth.select}} * 0.0975);
+                      height: calc({{$ctrl.credentialWidth.select}} * 0.0975)">
```

Comment:
Fix indentation. Also, CSS `calc` wouldn't have to be relied upon if these calculations were performed in the controller and stored in some object for the info overlay. The factor `0.0975` could also be in the controller as a constant. Also, why is that multiplier sometimes `0.0975` and `0.10` at other points? Can something be done to make that seem less magical and easier to understand why it is what it is?